tool/net/redbean.c

```diff
@@ -6472,9 +6471,9 @@ static void RestoreApe(void) {
   if (IsOpenbsd()) return; /* TODO */
   if (IsNetbsd()) return;  /* TODO */
   if (endswith(zpath, ".com.dbg")) return;
-  close(zfd);
-  zfd = OpenExecutable();
   if ((a = GetAssetZip("/.ape", 5)) && (p = LoadAsset(a, &n))) {
+    close(zfd);
+    zfd = OpenExecutable();
```

Comment:
While this works for my use case (redbean starts successfully now for me with this change), I am concerned I might be breaking something else I'm not aware of.